src/pages/newplant.js

```diff
@@ -163,15 +160,11 @@ export default function NewPlant() {
 					<div className='selectedPlant'>
 						<div className='context mt-2 has-text-weight-bold'></div>
 						<ul>
-							{Object.entries(getPlant).map(function (el) {
-								const [key, value] = el
-								return (
-									<li key={key}>
-										<span className='context has-text-weight-bold'>{key}</span>{' '}
-										: {value}
-									</li>
-								)
-							})}
+							<li>Species: {getPlant['species']}</li>
```

Comment:
We decided we were just displaying species and commonName here, not the extras light, water, etc.